packages/site/src/components/ComponentLinks.tsx

```diff
@@ -9,16 +10,68 @@ import { useAtlantisSite } from "../providers/AtlantisSiteProvider";
  */
 export const ComponentLinks = ({
   links,
+  goToProps,
+  goToUsage,
+  goToDesign,
 }: {
   readonly links?: ContentExportLinks[];
+  readonly goToProps: () => void;
+  readonly goToUsage: () => void;
+  readonly goToDesign: () => void;
 }) => {
+  const [hlinks, setHlinks] = useState<Element[] | null>(null);
+
+  useEffect(() => {
+    const hdd = document.querySelectorAll("[data-heading-link]");
+
+    if (hdd.length > 0) {
+      setHlinks(Array.from(hdd));
+    }
+  }, []);
+
+  const click = (e: MouseEvent) => {
+    e.preventDefault();
+    const id = e.currentTarget?.getAttribute("href")?.replace("#", "");
+
+    if (id) {
+      goToDesign();
+      setTimeout(() => {
+        const element = document.getElementById(id);
+
+        if (element) {
+          element.scrollIntoView({ behavior: "smooth" });
+        }
+      }, 100);
+    }
+  };
   const { isMinimal } = useAtlantisSite();
   if (isMinimal) return null;
 
   return (
     <Content>
       <Box padding="base" direction="column">
-        <Heading level={2}>Links</Heading>
+        <Box margin={{ bottom: "base" }}>
+          <Heading level={3}>Design</Heading>
+          {hlinks?.map((link, index) => (
+            <Box key={index}>
+              <a onClick={click} href={`#${link.id}`}>
+                {link.textContent}
+              </a>
+            </Box>
+          ))}
+        </Box>
+        <Box margin={{ bottom: "base" }}>
```

Comment:
I don't think these `Box`es need a margin if we wrap the lot of em in a `Content`

Reply:
Overall structure is 

`base` spacing (ie default `Content`)between headers and links in a section

`larger` spacing (ie `Content spacing="larger"`) between sections

smaller spacing between Icon and Link in the "external" Links section (maybe `Flex direction="horizontal" template={["shrink", "grow"]} gap="smaller"`?)

Reply:
For the spacing between Links, looks like we might only need to add `smaller` (Figma in red, screenshot of rendered in green)
<img width="276" alt="image" src="https://github.com/user-attachments/assets/16b9da85-8fe4-4225-81ec-eb3222dba1a4">

Reply:
I THINK I've addressed this, but I didn't compare it back to the Figma. So let's sit down for 10 if it's still off and we'll adjust together until it's good to go.